chore: Improve articles about Git repositories URLs usage (#2642)
codenvy/che-docs: modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc

[id="starting-a-new-workspace-from-a-devfile-url"]
= Starting a new workspace from a devfile URL

[IMPORTANT]
====

> To initiate a clone of the Git repository in the filesystem of the new workspace, the devfile must contain project info.

Reviewer: ```suggestion
To initiate a clone of the Git repository in the filesystem of a new workspace, the devfile must contain project info.
```

See https://devfile.io/docs/2.2.0/adding-projects.
====

With {prod-short}, you can open a *devfile* URL in your browser to start a new workspace.

pass:[<!-- vale RedHat.Spelling = NO -->]

> TIP: You can also use the *Git Repo URL* field on the *Create Workspace* page of your {prod-short} dashboard to enter the URL of a *devfile* to start a new workspace.

Reviewer: ```suggestion
TIP: You can use the *Git Repo URL* field on the *Create Workspace* page of your {prod-short} dashboard to enter the URL of a *devfile* to start a new workspace.
```

+
[TIP]
====
You can pass your personal access token to the URL to access a devfile from *private* repositories:
[source,subs="+quotes,+attributes,+macros"]
----
pass:c,a,q[{prod-url}]#__https://__<token>__@__<host>__/__<path_to_devfile> <1>
----
<1> Your personal access token that you generated on the Git provider's website.

This works for GitHub, GitLab, Bitbucket, Microsoft Azure, and other providers that support Personal Access Token.
====
+
[IMPORTANT]
====

> The automated git credential injection is not supported.

Reviewer: ```suggestion
Automated Git credential injection is not supported.
```

> To configure the git credentials, use xref:using-a-git-provider-access-token.adoc[configure personal access token] guide.

Reviewer: ```suggestion
To configure the Git credentials, use the xref:using-a-git-provider-access-token.adoc[configure personal access token] guide.
```

====
+
.Verification

> After you enter the URL to start a new workspace in a browser tab, it renders the workspace-starting page. When the new workspace is ready, the workspace IDE loads in the browser tab.

Reviewer: ```suggestion
After you enter the URL to start a new workspace in a browser tab, the workspace starting page appears. When the new workspace is ready, the workspace IDE loads in the browser tab.
```
